tldw_Server_API/tests/External_Sources/test_sync_coordinator.py

```
@@ -143,6 +144,96 @@ async def test_reconcile_created_change_creates_media_binding_and_initial_versio
+    _, source = await _create_account_and_source(connectors_db)
+    calls: list[dict[str, object]] = []
+
+    class _FakeMediaRepository:
+        def add_media_with_keywords(self, **kwargs):
+            calls.append(kwargs)
+            return 77, "repo-created", "created"
+
+    monkeypatch.setattr(
+        sync_coordinator_module,
+        "get_media_repository",
+        lambda media_db: _FakeMediaRepository(),
```

Ruff (0.15.6)

[warning] Unused lambda argument: `media_db`

(ARG005)